src/ril_voicecall.c

```diff
@@ -581,16 +581,37 @@ static void ril_voicecall_submit_hangup_req(struct ofono_voicecall *vc,
 {
 	struct ril_voicecall *vd = ril_voicecall_get_data(vc);
 	GRilIoRequest *ioreq = grilio_request_array_int32_new(1, id);
+        GSList *l;
+        struct ofono_call *call = NULL;
+	guint code;
 
 	/* Append the call id to the list of calls being released locally */
 	GASSERT(!gutil_int_array_contains(vd->local_release_ids, id));
 	gutil_int_array_append(vd->local_release_ids, id);
 
+        /* Look for call with id to get its status. */
+        for (l = vd->calls; l; l = l->next) {
+            call = l->data;
+            if (call->id == id)
+                break;
+            call = NULL;
+        }
```

Comment:
I think there's some space/tab indentation discrepancy here.

Reply:
Ah, good point. Fixed, thank you.